ONSdigital/census-rm-case-service
# updated the tag name of the image

# Motivation and Context
When building locally the image is currently census-rm-casesvc, docker-dev looks for eu.gcr.io/census-rm-ci/rm/census-rm-casesvc.  So having this changes makes using locally easier

# What has changed
Change to tagging in pom.xml

# How to test?
Local copy, mvn clean install.
Then:  docker ps | grep casesvc
The latest image should be eu.gcr.io/census-rm-ci/rm/census-rm-casesvc

# Links
https://trello.com/c/8sVc2ua0/624-fix-built-image-name-for-casesvc
